### PR TITLE
Filter :repl-env key from exception data

### PR DESCRIPTION
**Pull request by cichli**
Fixes clojure-emacs/cider#1004.

### CODE REVIEW

File: src/cider/nrepl/middleware/stacktrace.clj [hunk 1]
```diff
@@ -108,6 +108,13 @@
                #".* (compiling:)\((.*)\)" "Error $1 $2")
     cause))
 
+(def ^:private ex-data-blacklist #{:repl-env})
```
Comment (bbatsov) [inline src/cider/nrepl/middleware/stacktrace.clj]: Please, add some comment here for posterity's sake.

**Comment by expez** [comment]
That bot is pretty annoying, can we shut him up?  No single PR is likely to move the coverage statistics in a meaningful way.

**Comment by cichli** [comment]
@bbatsov yeah, of course - done.

@expez the bot can be disabled on https://coveralls.io/r/clojure-emacs/cider-nrepl (bottom left), it seems redundant when it's already using the status API.

**Comment by bbatsov** [comment]
:+1: 

P.S. I've disabled the comments.

**Comment by cichli** [comment]
I'll add a note to the client change log this evening too, since I've seen at least one person mention the problem on IRC.

**Comment by bbatsov** [comment]
Sure.

On 12 March 2015 at 12:06, Michael Griffiths <notifications@github.com>
wrote:

> I'll add a note to the client change log this evening too, since I've seen
> at least one person mention the problem on IRC.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/clojure-emacs/cider-nrepl/pull/169#issuecomment-78452674>
> .
>



-- 
Best Regards,
Bozhidar Batsov

http://www.batsov.com

**Pull request merged**